packages/component-submission/server/resolvers/index.js

```diff
@@ -1,7 +1,7 @@
 const logger = require('@pubsweet/logger')
-const { User } = require('@elifesciences/component-model')
+const User = require('@elifesciences/component-model-user')
 const Manuscript = require('@elifesciences/component-model-manuscript')
-const elifeApi = require('@elifesciences/component-model/entities/user/helpers/elife-api')
+const elifeApi = require('@elifesciences/component-model-user/entities/user/helpers/elife-api')
```

Comment:
can not `elifeApi` be exported and then de-structured?

Reply:
I think we should leave it as is so that User is the same as the other models and we can discuss what to do with the elifeApi later and raise a separate ticket for it.